13.technical-support/02.Adware/docs.en.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Adware/Viruses
+published: true
+taxonomy:
+    category:
+        - docs
+---
+
+If you suspect your computer is infected by malware or viruses, first of all you need to check your computer with free utilities from leading antiviruses.
+
+**Free antivirus utility**
+
+[Kaspersky Virus Removal Tool](https://www.kaspersky.ru/downloads/thank-you/free-virus-removal-tool)
+
+[Dr.Web CureIt! ](http://www.freedrweb.com/cureit/?lng=en)
+
+If this check with antivirus utilities revealed nothing, but the computer still behaves suspiciously, try to test it for adware, which can implement its advertising on any pages that you open.
+
+## How to clean your system from Adware?
+
+In the Internet, you can find a lot of applications that are aimed at fighting this problem. You can use any utility you like. But we strongly recommend using **[AdwareCleaner](https://downloads.malwarebytes.com/file/adwcleaner)** from **Malwarebytes**. This application is quite easy to use and you can test it by yourself.
+
+After the application has been downloaded and successfully installed, you should take the following steps:
+
+1) Start the application and tap **scan now**.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Windows7.1/adware1.png" />
+
+2) Once the scan has been completed, you will need to send suspicious objects to **quarantine**.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Windows7.1/adware2.png" />
+
+3) When the items are moved to **quarantine**, you will be able to **clear the system** of unnecessary ballast.
+4) In some cases, you may need to **reboot the system** after cleaning is complete.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Windows7.1/adware3.png" />
+
+5) Once the system has been rebooted, cleaning will be completed, if necessary, you can always read the **report** on the work done.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Windows7.1/adware4.png" />
+
+If it didn’t help - disable all settings in the browser in which this situation occurred. Use [this instructions](#instruction). 
+
+If disabling of settings and extensions does not help, try to [re-create shortcuts of your browsers](#shortcuts).
+
+Below on this page we’ve described the instructions for removing some of the most popular ad extensions.
```

Comment:
Further on this page we describe the instructions to remove some of the most commonly encountered ad extensions.